internal/pkg/telemetry/windows_cpu.go

```diff
@@ -0,0 +1,84 @@
+// +build windows

```

Comment:
You're going to need to exclude Windows from `unimplented_cpu`.

Reply:
Yup makes sense, will do.